sass/site/primary/_posts-and-pages.scss

```diff
@@ -412,7 +412,7 @@
 		.entry-thumbnail {
 			float: left;
 			margin: 0;
-			width: 42.452830188679%;
```

Comment:
Yeah. Figured out the problem. I was not using a sidebar, so there was a large gap between the image and content. And i'm guessing you are using a setup with an active sidebar. 

To sort out this problem, what we can do set two different widths for search results with and without sidebars.